k256/src/arithmetic/projective.rs

```diff
@@ -346,6 +444,15 @@ impl<'a> Neg for &'a ProjectivePoint {
     }
 }
 
+// TODO(tarcieri): double check how this should be implemented
+impl Display for ProjectivePoint {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        write!(f, "{:X}", self.x.to_bytes())?;
+        write!(f, "{:X}", self.y.to_bytes())?;
+        write!(f, "{:X}", self.z.to_bytes())
+    }
+}
```

Comment:
What should this `Display` impl actually look like?

Reply:
This seems like a good case for a [debug struct](https://doc.rust-lang.org/stable/std/fmt/struct.Formatter.html#method.debug_struct).

So maybe, it could look something like this: 
```rust
 fmt.debug_struct("ProjectivePoint")
        .field("x point: {:X}", self.x.to_bytes())
        .field("y point: {:X}", self.y.to_bytes())
        .field("z point: {:X}", self.z.to_bytes())
        .finish()
```

Reply:
This is a `Display` impl though, so I'd expect it to give something easily parsable.

Looking at `bls12_381` though, `Display` is impl'd equivalently to `Debug`, so I guess I'll just do that.

Reply:
Updated to use the same impl as `bls12_381`